src/renderers/webgl/WebGLBackground.js

```diff
@@ -105,10 +109,24 @@ function WebGLBackground( renderer, cubemaps, cubeuvmaps, state, objects, alpha,
 
 			}
 
+			_e1.copy( scene.backgroundRotation );
+
+			_e1.x *= - 1;
+
+			if ( ( background.isCubeTexture && background.isRenderTargetTexture === true ) || background.mapping === CubeUVReflectionMapping ) {
+
+				// cube render targets and cubeUV maps (PMREM) need to reverse Y and Z rotation (which have a different conventions than normal cube textures, see flipEnvMap)
+
+				_e1.y *= - 1;
+				_e1.z *= - 1;
+
+			}
```

Comment:
All three components must be negated to accommodate the left-handed coordinate frame.

Subsequently, the non-render-targets are handled as a special case -- as we have in the past.

```js
_e1.copy( material.envMapRotation );

_e1.negate(); // accommodate left-handed frame

if ( background.isCubeTexture && background.isRenderTargetTexture === false )

	_e1.x *= - 1;

}
```
This makes more sense to me. Maybe you can confirm this conjecture is correct?

Reply:
Perfect! I just had to inline `negate()` because `_e1` is of type `Euler`.